ArmoniK.Extension.CSharp.Client/Services/EventsService.cs

```diff
@@ -38,6 +38,19 @@ internal class EventsService : IEventsService
 
   private readonly ILogger<EventsService> logger_;
 
+  /// <summary>
+  ///   Creates an instance of <see cref="IEventsService" /> using the specified GRPC channel and an optional logger
```

Comment:
```suggestion
  ///   Creates an instance of <see cref="EventsService" /> using the specified GRPC channel and an optional logger
```

Should be done for the others too. Interfaces cannot be instanciated and return parameters are not valid for constructors.

Reply:
By the way, I'd split documentation from this PR that adds DI